examples/object-type-reference/object_type_reference.bal

```diff
@@ -38,8 +38,11 @@ type Manager object {
 
     public string dpt;
 
+    // referenced fields can be overridden by subtypes
+    public float salary;
```

Comment:
If the salary is a `float`, the `init` function should assign a `float` value to the `salary` field, right?

Reply:
Fixed : a0fb6d56bd8b7475a57a4228932fc8ba58c28396